### PR TITLE
cachetool recipe: add cachetool_bin setting to download old version (…

### PR DESCRIPTION
**Pull request by hyde1**
…4.0.0 is only compatible with php >=7.1)

| Q             | A
| ------------- | ---
| Bug fix?      | Yes
| New feature?  | No
| BC breaks?    | No
| Deprecations? | No
| Fixed tickets | #183

> Do not forget to add notes about your changes to [CHANGELOG.md](https://github.com/deployphp/recipes/blob/master/CHANGELOG.md)

### CODE REVIEW

**Comment by antonmedv** [comment]
Why just don't update phar?

**Comment by antonmedv** [comment]
You proper phar tool :)

File: recipe/cachetool.php [hunk 1]
```diff
@@ -8,6 +8,9 @@
 namespace Deployer;
 
 set('cachetool', '');
+set('cachetool_bin', function(){
+	return run("/usr/bin/php -r \"echo version_compare(phpversion(), '7.1') == -1 ? 'cachetool-3.2.1.phar' : 'cachetool.phar';\"");
```
Comment (antonmedv) [inline recipe/cachetool.php]: Where is _cachetool-3.2.1.phar_ should come?

**Comment by hyde1** [comment]
It is hosted a the same place as cachetool.phar :)

File: recipe/cachetool.php (hunk 1 above)
Comment (antonmedv) [reply recipe/cachetool.php]: It's better to use `{{bin/php}}` instead of `/usr/bin/php`.

File: recipe/cachetool.php [hunk 2]
```diff
@@ -8,6 +8,9 @@
 namespace Deployer;
 
 set('cachetool', '');
+set('cachetool_bin', function(){
```
Comment (antonmedv) [inline recipe/cachetool.php]: Rename to `bin/cachetool`

**Comment by hyde1** [comment]
Sure, idk why I hardcoded that...

**Comment by hyde1** [comment]
Done!

**Pull request merged**